### ISSUE TITLE
tasks don't use STATIC_ROOT

### ISSUE THREAD

**Issue by oppianmatt** [kmmbvnr/django-jenkins#270]
Was wondering why run_csslint and run_jshint doesn't look at the STATIC_ROOT.

It get's the list like so:

```
        files = list(
            static_files_iterator(apps_locations + list(getattr(settings, 'STATICFILES_DIRS', [])),
                                  extension='.css',
                                  ignore_patterns=options['csslint_exclude'].split(',')))
```

apps_locations is the list of either INSTALLED_APPS or PROJECT_APPS, and STATICFILES_DIRS is the additional directories, but it doesn't add STATIC_ROOT. We can't add STATIC_ROOT to STATICFILES_DIRS as django throws an error. We can't also specify extra paths to search for with the tasks.

**Comment by kmmbvnr** [comment]
Looks like you don't understand the STATIC_ROOT settings meaning. Please, look for not django docs.

**Comment by oppianmatt** [comment]
I would argue that we should check STATIC_ROOT also or an option to check it (or be able to pass a custom directory to the linters, like include dir, we have exclude). The docs state that's where all the collectstatic files will go. So as part of a deploy/build the statics have all gone there and then jenkins runs the css/js linters on the final statics that are about to go out. If people haven't run collectstatic then the dir is empty and linters run on the STATICFILES_DIRS anyway. So shouldn't be an issue either way.

Basically all I'm asking for is the ability to add a directory to the linters like an include. We have exclude and then we can pass in STATIC_ROOT as part of the deploy check.
